### PR TITLE
hit-1170300223

### PR DESCRIPTION
**Pull request by Laurence-042**
Modified the readme file to English.

### CODE REVIEW

**Comment by 1170300405** [comment]
*Try to improve it with pr!*
请把这部分改为黑体。

**Review by 1170300405 — CHANGES_REQUESTED** [review]
Try to improve it with pr!
请把这部分改为黑体。

**Review by Laurence-042 — COMMENTED** [review]
OK

File: README.md [hunk 1]
```diff
@@ -1,2 +1,11 @@
...
+This project is a clock simulating program。 Simulated the function of the clock.
+
+## Using
+Run it in Eclipse, or simply use `javac` to compile and `java` to run.
+
+## On the way
+Trying to make a GUI to make it easier to use and build with maven to make it easier to build.
+
+## And more
+**Try to improve it with pr!**
```
Comment (Laurence-042) [inline README.md:11]: OK

Comment (Laurence-042) [inline README.md:11]: Bold

**Comment by 1170300405** [comment]
OK

**Pull request merged**